docs/ch_running.adoc

```diff
@@ -8,6 +8,10 @@ go and read Leo Brodie's excellent "Starting Forth", available for
 free online, before continuing through this manual.  You can use Tali
 Forth 2 to run the examples given in the book.
 
+You will also want to refer to Tali's <<Glossary>> for the list of words that
```

Comment:
It wasn't clear to me in the TOC that "Glossary" referred to forth words, rather than (say) a glossary of concepts in the manual.   What about calling it "Glossary of Built in Forth Words" or some such?

Reply:
That's a good point (and another one where I was blind to the issue - if I'm working with a new forth, it's glossary is the first thing I look for).  I hadn't even thought about the fact that the word glossary has multiple meanings here, and that either type could be expected in a document like this.